src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/IPFilterRules/RemoveAzureEventHubIpFilterRule.cs

```diff
@@ -16,39 +16,33 @@
 using System.Management.Automation;
 using Microsoft.Azure.Commands.EventHub.Models;
 
-namespace Microsoft.Azure.Commands.EventHub.Commands.ConsumerGroup
+namespace Microsoft.Azure.Commands.EventHub.Commands.EventHub
 {
     /// <summary>
-    /// 'Remove-AzureRmEventHubConsumerGroup' deletes the specifed Consumer Group
+    /// 'Remove-AzureRmEventHub' Cmdlet removes the specified EventHub
     /// </summary>
-    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "EventHubConsumerGroup", DefaultParameterSetName = ConsumergroupPropertiesParameterSet, SupportsShouldProcess = true), OutputType(typeof(void))]
-    public class RemoveAzureRmEventHubConsumerGroupp : AzureEventHubsCmdletBase
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "EventHubIPFilterRule", DefaultParameterSetName = IpFilterRulePropertiesParameterSet, SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveAzureEventHubIpFilterRule : AzureEventHubsCmdletBase
     {
-        [Parameter(Mandatory = true, ParameterSetName = ConsumergroupPropertiesParameterSet, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
+        [Parameter(Mandatory = true, ParameterSetName = IpFilterRulePropertiesParameterSet, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
```

Comment:
@v-Ajnava same comment about the `ValueFromPipelineByPropertyName` property only being applied to `-ResourceId` in this cmdlet